db_store.py

```
@@ -407,13 +407,17 @@ def defrag_availability(db_conn=None):
-            # 1. Select all blocks sorted by id and start_time
-            c.execute(
-                f"SELECT id, {id_col}, start_time, end_time FROM {table} ORDER BY {id_col}, start_time"
-            )
+            # Use explicit queries for whitelisted tables to prevent SQL injection risk
+            if table == "crew_availability":
+                id_col = "crew_id"
+                c.execute(
+                    "SELECT id, crew_id, start_time, end_time FROM crew_availability ORDER BY crew_id, start_time"
+                )
+            else:
+                id_col = "appliance_id"
```

Ruff (0.15.1)

[error] Local variable `id_col` is assigned to but never used

Remove assignment to unused variable `id_col`

(F841)